src/Earnings/SFA.DAS.Earnings.Api/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.SharedOuterApi.Services;
+
+namespace SFA.DAS.Earnings.Api.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class ErrorController : ControllerBase
+{
+    private readonly ILogger<ErrorController> _logger;
+
+    public ErrorController(ILogger<ErrorController> logger)
+    {
+        _logger = logger;
+    }
+
+    [HttpGet]
+    [Route("")]
+    public IActionResult Error()
+    {
+        if (HttpContext.Features.Get<IExceptionHandlerFeature>()?.Error is ApiUnauthorizedException)
```

Comment:
Is it worth adding a unit test for this endpoint. It would establish a unit test project

Reply:
sorted